.phpunit.result.cache

```diff
@@ -0,0 +1 @@
+C:37:"PHPUnit\Runner\DefaultTestResultCache":446:{a:2:{s:7:"defects";a:3:{s:7:"Warning";i:6;s:78:"Tests\ServiceContainer\ServiceContainerTest::testAutoConfigurationCannotBeDone";i:4;s:75:"Tests\ServiceContainer\ServiceContainerTest::testAutoConfigurationCanBeDone";i:5;}s:5:"times";a:3:{s:7:"Warning";d:0.005;s:78:"Tests\ServiceContainer\ServiceContainerTest::testAutoConfigurationCannotBeDone";d:0.025;s:75:"Tests\ServiceContainer\ServiceContainerTest::testAutoConfigurationCanBeDone";d:0.001;}}}
```

Comment:
Could you add this file to `.gitignore`?

Reply:
Fixed